albyhub/docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
     environment:
       APP_HOST: albyhub_server_1
       APP_PORT: 8080
-      PROXY_AUTH_ADD: "false"
+      PROXY_AUTH_ADD: "true"
```

Comment:
Isn't _true_ the default here so you can omit the whole line? 

From the [docs](https://github.com/getumbrel/umbrel-apps/blob/840a1ef8ebb30c3de8378442b21e05108708762c/README.md?plain=1#L455C1-L455C105): 
> The Umbrel App Proxy automatically protects an app by requiring the user to enter their Umbrel password


```suggestion
```